app/views/summary/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1 class="govuk-heading-xl">
+  User intent survey
+</h1>
+
+<p class="govuk-body govuk-!-margin-bottom-7">Last updated 6 April 2020 9:04am</p>
```

Comment:
Is this date needed?  If it is, could it be generated somehow?  If you have to update this line every time you change this page, someone will eventually forget.

Reply:
This was added to be consistent with the design and for the UR sessions. At the moment the data is updated via importing from a CSV - not ideal. We could store the import metadata and use this to show an updated at datetime here. Definitely agree with your comment.